enigma-cli/src/main/java/cuchaz/enigma/command/HelpCommand.java

```diff
@@ -0,0 +1,27 @@
+package cuchaz.enigma.command;
+
+import org.tinylog.Logger;
+
+import java.util.Collection;
+
+public class HelpCommand extends Command {
+	protected HelpCommand() {
+		super("help");
+	}
+
+	@Override
+	public void run(String... args) throws Exception {
+		StringBuilder help = new StringBuilder();
+		Collection<Command> commands = Main.getCommands().values();
+
+		help.append("Supported commands:").append("\n");
+		for (Command command : commands) {
+			help.append("- ").append(command.name).append("\n");
+			// todo iota please help!
+			//help.append("\t").append(command.description.append("\n");
```

Comment:
@IotaBread I'd like you to write descriptions for the commands so we can do this :)

Reply:
wrote up descriptions on all the ones I'm familiar with, that leaves four more that I'd like help with